x/profiles/types/models_params.go

```diff
@@ -173,3 +190,76 @@ func ValidateBioParams(i interface{}) error {
 
 	return nil
 }
+
+// ___________________________________________________________________________________________________________________
+
+// NewOracleParams creates a new Oracle Params obj
+func NewOracleParams(
+	scriptID int64,
+	askCount,
+	minCount,
+	prepareGas,
+	executeGas uint64,
+	feePayer string,
+	feeAmount ...sdk.Coin,
+) OracleParams {
+	return OracleParams{
+		ScriptID:   scriptID,
+		AskCount:   askCount,
+		MinCount:   minCount,
+		PrepareGas: prepareGas,
+		ExecuteGas: executeGas,
+		FeePayer:   feePayer,
+		FeeAmount:  feeAmount,
+	}
+}
+
+func DefaultOracleParams() OracleParams {
+	return NewOracleParams(
+		DefaultOracleScriptID,
+		DefaultOracleAskCount,
+		DefaultOracleMinCount,
+		DefaultOraclePrepareGas,
+		DefaultOracleExecuteGas,
+		DefaultOracleFeePayer,
+		DefaultOracleFeeCoins...,
```

Comment:
I think we should delete the default values for `OracleScriptID`, `OracleFeePayer` and `OracleFeeCoins` as they should be set during genesis and not with default values

Reply:
The default values are used for `DefaultParams()` which is called by `DefaultGenesisState()`.  After removing, Params still need default values for `DefaultGenesisState()`. So, I think we should keep it.

Reply:
@dadamu We should set empty values where needed inside the `DefaultGenesisState` and that's all, no need to set them as publicly available constants